packages/integrations/image/README.md

```diff
@@ -359,6 +359,16 @@ The output formats to be used in the optimized image. If not provided, `webp` an
 
 For remote images, the original image format is unknown. If not provided, only `webp` and `avif` will be used.
 
+#### quality
```

Comment:
This documentation change is approved by Astro Docs Maintainers, if the feature itself is approved, and will be automatically reflected in Docs. No additional docs PR would be necessary!